aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -295,6 +303,10 @@ module aptos_framework::delegation_pool {
         delegate_voting_power_events: EventHandle<DelegateVotingPowerEvent>,
     }
 
+    struct Beneficiary has key {
+        beneficiary: address,
+    }
+
     struct AddStakeEvent has drop, store {
```

Comment:
how do you want to migrate to event v2? since you add new v2 events but don't migrate existing v1 events...

Reply:
This PR is not migrating all the existing V1 events, but a follow-up PR will do.